crates/biome_grit_formatter/tests/spec_test.rs

```diff
@@ -0,0 +1,27 @@
+use biome_formatter_test::spec::{SpecSnapshot, SpecTestFile};
+use biome_grit_formatter::{context::GritFormatOptions, GritFormatLanguage};
+use std::path::Path;
+
+mod language {
+    include!("language.rs");
+}
+
+pub fn run(spec_input_file: &str, _expected_file: &str, test_directory: &str, _file_type: &str) {
+    let root_path = Path::new(concat!(env!("CARGO_MANIFEST_DIR"), "/tests/specs/"));
+
+    let Some(test_file) = SpecTestFile::try_from_file(spec_input_file, root_path, None) else {
+        return;
```

Comment:
I recommend panicking in this else branch, otherwise the test would fail silently. This happened in the html formatter recently.

Reply:
Looks like that uncovered a bug! Working to address it. Thanks!